ci/images/google-geode-builder/scripts/setup.sh

```diff
@@ -57,7 +59,9 @@ DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
 cp -R /etc/alternatives /etc/keep-alternatives
 apt-get install -y --no-install-recommends \
     openjdk-8-jdk \
```

Comment:
Why open jdk and bellsoft on the same image?

Reply:
Last time we were bit by trying to move the entire mountain at once, and missing a few places (for example, PRs for support branches use this too, even other pipelines like examples, plus there's a lag time for images to rebuild, etc).

To avoid disruption, the plan this time is to get both JDKs in the image, then transition everything over and verify it's working, then finally remove the unneeded JDK.